packages/amplify-graphql-auth-transformer/src/graphql-auth-transformer.ts

```diff
@@ -808,6 +808,7 @@ export class AuthTransformer extends TransformerAuthBase implements TransformerA
         MappingTemplate.inlineTemplateFromString('$util.toJson({})'),
         NONE_DS,
         stack,
+        ctx.disableResolverDeduping,
       );
```

Comment:
Could this be injected in the `host` creation rather than for each appsync function addition? That should be a smaller footprint for change, and require us to plumb this through fewer layers that should have no knowledge of this concern.